### ISSUE TITLE
Replace wp1-dev-database with git-lfs

### ISSUE THREAD

**Issue by kelson42** [openzim/wp1#278]
That sounds more elegant to me.

**Comment by audiodude** [comment]
What exactly would be stored in LFS in this scenario? A docker image? A .sql dump file?

**Comment by kelson42** [comment]
The same as what you have in the git. Would recommend to consider to put a compressed version of it.

**Comment by audiodude** [comment]
The git repo for wp1-dev-database contains a Dockerfile and is used to build an image which is included in the `docker-compose-dev.yml` file:

https://github.com/openzim/wp1/blob/main/docker-compose-dev.yml#L11

But I agree, in general, that having the dev database in a separate repo is a bit confusing, especially as I consider the schema changes that are needed for selection.

**Comment by kelson42** [comment]
yes, if we use git-lfs, then this `Dockerfile` can advantageously be part of the testing code available in this repository. Then everything is in an atomic git repository.

**Comment by mahakporwal02** [comment]
@kelson42 @audiodude should git LFS be installed in wp1-dev-database and track .sql file in that repository only?

**Comment by audiodude** [comment]
@mahakporwal02 No, we want to get rid of the `wp1-dev-database` repo and instead store all of its contents in this `wp1` repo. But because the .sql file is large, we want to use git-lfs to store it.

Additionally, the `docker-compose-dev.yml` file that I linked above needs to be updated to build the dev database from a "local" Dockerfile that is in a subdirectory instead of pulling it from docker hub.

**Comment by audiodude** [comment]
I'm actually going to take this issue because it's blocking my plan for schema migrations and I want to start working on that

**Comment by mahakporwal02** [comment]
> @mahakporwal02 No, we want to get rid of the `wp1-dev-database` repo and instead store all of its contents in this `wp1` repo. But because the .sql file is large, we want to use git-lfs to store it.
> 
> Additionally, the `docker-compose-dev.yml` file that I linked above needs to be updated to build the dev database from a "local" Dockerfile that is in a subdirectory instead of pulling it from docker hub.

OK, thanks
